airflow-core/src/airflow/api_fastapi/core_api/datamodels/dag_run.py

```diff
@@ -168,3 +168,11 @@ class DAGRunsBatchBody(StrictBaseModel):
     start_date_lte: AwareDatetime | None = None
     end_date_gte: AwareDatetime | None = None
     end_date_lte: AwareDatetime | None = None
+    run_after_gt: AwareDatetime | None = None
+    run_after_lt: AwareDatetime | None = None
+    logical_date_gt: AwareDatetime | None = None
+    logical_date_lt: AwareDatetime | None = None
+    start_date_gt: AwareDatetime | None = None
+    start_date_lt: AwareDatetime | None = None
+    end_date_gt: AwareDatetime | None = None
+    end_date_lt: AwareDatetime | None = None
```

Comment:
nit: Maybe sort this, so xxx_gt, xxx_gte, xxx_lt, xxx_lte and up together, it will be easier to read